modules/server/src/test/scala/higherkindness/compendium/db/DBServiceStub.scala

```diff
@@ -17,10 +17,18 @@
 package higherkindness.compendium.db
 
 import cats.effect.IO
+import cats.effect.concurrent.Ref
+import higherkindness.compendium.models.{IdlNames, MetaProtocolDB}
 import higherkindness.compendium.core.refinements.ProtocolId
 
-class DBServiceStub(val exists: Boolean) extends DBService[IO] {
-  override def upsertProtocol(id: ProtocolId): IO[Unit]    = IO.unit
-  override def existsProtocol(id: ProtocolId): IO[Boolean] = IO.pure(exists)
-  override def ping(): IO[Boolean]                         = IO.pure(exists)
+class DBServiceStub(val exists: Boolean, refProt: Option[Ref[IO, MetaProtocolDB]] = None)
```

Comment:
Why do you need to use `Ref`... I'm not totally sure, but I think we could avoid it :thinking:

Reply:
Yes, you're right, I forgot to change it, thanks! 😄